concent_api/gatekeeper/views.py

```diff
@@ -121,73 +157,193 @@ def parse_headers(request: WSGIRequest, path_to_file: str) -> Union[FileTransfer
     # Check ConcentFileTransferToken each field:
     # -DEADLINE
     if not isinstance(loaded_golem_message.token_expiration_deadline, int):
-        return gatekeeper_access_denied_response('Wrong type of token_expiration_deadline variable.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'Wrong type of token_expiration_deadline variable.',
+            constants.ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
     current_time = get_current_utc_timestamp()
 
     if current_time > loaded_golem_message.token_expiration_deadline:
-        return gatekeeper_access_denied_response('token_expiration_deadline has passed.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'token_expiration_deadline has passed.',
+            constants.ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
 
     # -STORAGE_CLUSTER_ADDRESS
     if not isinstance(loaded_golem_message.storage_cluster_address, str):
-        return gatekeeper_access_denied_response('Wrong type of storage_cluster_address variable.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'Wrong type of storage_cluster_address variable.',
+            constants.ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
     url_validator = URLValidator()
     try:
         url_validator(loaded_golem_message.storage_cluster_address)
     except ValidationError:
-        return gatekeeper_access_denied_response('storage_cluster_address is not a valid URL.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'storage_cluster_address is not a valid URL.',
+            constants.ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE,
+            path_to_file,
+            loaded_golem_message.subtask_id,
+            client_public_key,
+        )
     if loaded_golem_message.storage_cluster_address != settings.STORAGE_CLUSTER_ADDRESS:
-        return gatekeeper_access_denied_response('Given storage_cluster_address is not defined in settings.', path_to_file, loaded_golem_message.subtask_id, client_public_key)
+        return gatekeeper_access_denied_response(
+            'Given storage_cluster_address is not defined in settings.',
+            constants.ERROR_CODE_GATEKEEPER_INVALID_REQUEST_MESSAGE,
```

Comment:
This should have a separate code. There's a possibility that the cluster address was valid before but has been removed. We want to be able to distinguish this from obvious errors like a broken token.

In general codes for errors caused by invalid data should be distinguishable from codes for data that is syntactically correct but is rejected for some semantic reason.